admin-console/src/main/java/com/agiletec/apsadmin/category/CategoryAction.java

```diff
@@ -260,31 +271,29 @@ protected void extractReferencingObjects(String categoryCode) {
 			_logger.error("Error extracting referenced objects by category {}", categoryCode, t);
 		}
 	}
-	
+
 	public String save() {
 		String parentCategoryCode;
 		try {
-			if(this.getParentCategoryCode().contains(",") && !StringUtils.endsWith(this.getParentCategoryCode(), ",")) {
+			if (this.getParentCategoryCode().contains(",") && !StringUtils.endsWith(this.getParentCategoryCode(), ",")) {
 				parentCategoryCode = StringUtils.trim(this.getParentCategoryCode().split(",")[1]);
 			} else {
 				parentCategoryCode = this.getParentCategoryCode();
 			}
 			if (this.getStrutsAction() == ApsAdminSystemConstants.EDIT) {
 				Category category = this.getCategory(this.getCategoryCode());
 				category.setTitles(this.getTitles());
-				category.setParentCode(parentCategoryCode);
+				//category.setParentCode(parentCategoryCode);
 				this.getCategoryManager().updateCategory(category);
 				_logger.debug("Updated category {}", category.getCode());
+			} else if (!StringUtils.isEmpty(this.getParentCategoryCode())) {

```

Comment:
we already defined a variable containing the parentCategoryCode, why do we have to go through the getter again?
btw why is the category a CSV?